src/guc.c

```diff
@@ -283,6 +284,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable("timescaledb.enable_ddl_on_global_objects",
+							 "Enable distributed DDL operations on global objects",
+							 "Forward DDL operations on global objects to data nodes",
```

Comment:
I think it would be good to elaborate in the long description on what we mean with a global object. I don't think it is clear for a user what a "global object" is.